src/frontend/screens/PhotosModal/index.tsx

```diff
@@ -1,24 +1,50 @@
-// @flow
-import React, { useState } from "react";
+import React, { useEffect, useRef, useState } from "react";
 import { Dimensions, View, StyleSheet } from "react-native";
 import { TabView } from "react-native-tab-view";
 
 import IconButton from "../../sharedComponents/IconButton";
+import Button from "../../sharedComponents/Button";
 import { CloseIcon } from "../../sharedComponents/icons";
 import { filterPhotosFromAttachments } from "../../lib/utils";
 import useObservation from "../../hooks/useObservation";
 import useDraftObservation from "../../hooks/useDraftObservation";
 import PhotoView from "../../sharedComponents/PhotoView";
 import api from "../../api";
 import type { NavigationProp } from "../../types";
-
+import { useBottomSheetModal } from "../../sharedComponents/BottomSheetModal";
+import { ConfirmDeleteModal } from "./ConfirmDeleteModal";
 const PhotosModal = ({ navigation }: { navigation: NavigationProp }) => {
   const observationId = navigation.getParam("observationId");
   const [index, setIndex] = useState(navigation.getParam("photoIndex") || 0);
   const [{ observation }] = useObservation(observationId);
   const [{ photos: draftPhotos }] = useDraftObservation();
+  const [showHeader, setShowHeader] = useState(false);
+  const { sheetRef, openSheet, closeSheet } = useBottomSheetModal({
+    openOnMount: false,
+  });
+  const modalIsOpen = useRef<boolean>(false);
+
+  //We don't want to show the header when the modal is open, these functions
+  //check if the modal is open before trying to show header
+  function toggleShowHeader() {
+    if (modalIsOpen.current) {
+      setShowHeader(false);
+      return;
+    }
+    setShowHeader(!showHeader);
```

Comment:
nit but I generally prefer to use the functional syntax for "toggle" interactions i.e. `setShowHeader(previous => !previous)`